src/uni_modules/wot-ui/components/wd-cell/wd-cell.vue

```
@@ -33,7 +33,7 @@
           <!--title END-->
 
           <!--label BEGIN-->
-          <slot name="label">
+          <slot v-if="(useLabelSlot && $slots.label)" name="label">
```

ESLint

[error] Replace `(useLabelSlot·&&·$slots.label)` with `useLabelSlot·&&·$slots.label`

(prettier/prettier)